tests/e2e/data/utils.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+from typing import Any, Iterator, List, Tuple
+
+from ..conftest import run_cli
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_tested_archive_types() -> List[str]:
+    env = os.environ.get("PYTEST_DATA_COPY_ARCHIVE_TYPES")
+    if env:
+        return env.split(",")
+    else:
+        return [".tar.gz"]
+
+
+def _run_command(cmd: str, args: List[str]) -> Tuple[int, str, str]:
+    logger.info(f"Running: {[cmd] + args}")
+    result = run_cli([cmd] + args)
+    return result.returncode, result.stdout, result.stderr
```

Comment:
it would require to pass it into other fixtures (many places where we run resource creation and existence checks), and would be rather redundant since it does not manage any environment and has no lifecycle useful for testing. IMO - using it as a fixture would yield no additional benefits

Reply:
agreed